src/components/KonnectorEdit.jsx

```diff
@@ -109,6 +109,7 @@ export const KonnectorEdit = ({ t, account, connector, deleting, disableSuccessT
               onSubmit={onSubmit}
               submitting={submitting}
               values={account ? account.auth || account.oauth : {}}
+              disableFolderPath
```

Comment:
Isn't something that should be into `AccountLoginForm`'s state ?

Reply:
This point is used only to disable it on edition modal (different from display or not advanced options). Or maybe I didn't see what you mean here.